ui.py

```diff
@@ -5,4 +5,24 @@
 
 UI passes the calander object to the API Manager and the API Manager will make calls
 to the three APIs using that date. 
-'''
\ No newline at end of file
+'''
+from flask import Flask, render_template, request
+from API_Movie_Call import movie_call
+
+app = Flask(__name__)
+
+@app.route('/')
+def homepage():
+    return render_template('home.html')
+
+@app.route('/get_date')
+def get_date_info():
+    print('form date is: ', request.args)
+    date_to_search = request.args.get('search_date')
+    movie_info = movie_call(date_to_search)
+    return render_template('api_results.html', movie_info=movie_info)
+
+
+if __name__ == '__main__':
+    app.run()
+## TODO Watch videos on Flask, set up a basic input. 
```

Comment:
Videos - are these youtube videos?

Reply:
That TODO was left over from before I started. It was reminding me about the Flask videos on the class website.

Reply:
Ah, that makes sense. I think I'm mixing up all the different features of all the different projects, or I was thinking that sometimes the APOD API returns a video instead of a picture. Or something.  Please disregard this comment.